source/tools/testConsistency/dotnetWPF-renderer/test_app.py

```diff
@@ -0,0 +1,152 @@
+# Copyright (c) Microsoft Corporation. All rights reserved. 
+# Licensed under the MIT License. 
+import unittest
+import json
+from app import app
+from copy import deepcopy
+
+
+POST_URL = 'http://127.0.0.1:5000/'
+TEST_PARTIAL_RESULT_URL = 'http://127.0.0.1:5000/test'
+
+class TestFlaskApi(unittest.TestCase):
+    def setUp(self):
+        #self.backup_items = deepcopy(app.items)  # no references!
```

Comment:
> #self.backup_items = deepcopy(app.items)  # no references! [](start = 7, length = 59)

Why is this commented out again? Ideally avoid commented out source code; If you haev to then add a comment explaining the comment :)